temboard/README.md

```diff
@@ -1,36 +1,59 @@
-Temboard images for docker
-==========================
+# Temboard images for docker
 
-This folder contains docker images for
-[temboard](https://github.com/dalibo/temboard).
+This folder contains docker images for [temboard](https://temboard.io/).
 
-A docker-compose file is also provided.  It's aimed to quickly try temboard
-with a few postgres servers.
 
-The docker-compose file will launch:
+## Quickstart
 
-  * a PG 9.4 cluster, configured to also run
-    [PoWA](http://dalibo.github.io/powa/)
-  * a PG 9.5 cluster, configured to also run
-    [PoWA](http://dalibo.github.io/powa/)
-  * a PG 9.6 cluster, configured to also run
-    [PoWA](http://dalibo.github.io/powa/)
-  * a standard PG 9.6 cluster for the UI
-  * a container for the UI
+A `docker-compose.yml` file is provided. It's aimed to quickly try temboard with
+a few postgres servers.
+
+``` console
+wget https://raw.githubusercontent.com/dalibo/docker/master/temboard/docker-compose.yml
+docker-compose up
+```
+
+The `docker-compose` file will launch:
+
+- a PG 9.4 cluster
+- a PG 9.5 cluster
+- a PG 9.6 cluster
+- a temboard agent for each PG cluster
+- a standard PG 9.6 cluster for the UI
+- a container for the UI
 
 After starting the docker-compose image, the UI is available on
+https://0.0.0.0:8888/ with `admin` / `admin` credentials.
+
+The agents can be accessed with `alice` / `alice` or `bob` // `bob`.
+
+
+## Warnings
+
+**DO NOT USE THIS IN PRODUCTION**
+
+Obviously these docker images are designed for testing and demo. All database
+access are set to ``trust``, the SSL certificate is self-signed and the default
+passwords are public.
+
+Temboard agent is designed to run on same host of PostgreSQL which is
+incompatible with Docker service-minded architecture. Temboard agent images
+requires access to docker socket to restart PostgreSQL. Which you do not want in
```

Comment:
Sorry, I should be more explicit:
"Temboar agent images requires" -> "Temboard agent images require" (images is plural)